### ISSUE TITLE
[rocm-opencl-runtime] clinfo doesn't work

### ISSUE THREAD

**Issue by tpkessler** [rocm-arch/rocm-arch#97]
The loader file in `/etc/OpenCL/vendors` is missing. Fixed with PR #96

**Comment by tpkessler** [comment]
Ok. Not really fixed with PR #96 / #97
`clinfo` shows me half of its output and then aborts with a build error:
```
$ /opt/rocm/bin/x86_64/clinfo
LoadLib(libhsa-ext-finalize64.so.1) failed: libhsa-ext-finalize64.so.1: cannot open shared object file: No such file or directory
ERROR: clBuildProgram(-11)
```
The same for `clinfo` from the upstream packge:
```
$ /usr/bin/clinfo
=== CL_PROGRAM_BUILD_LOG ===
Error: Failed to compile opencl source (from CL or HIP source to LLVM IR).
```

**Comment by tpkessler** [comment]
I've opened an issue upstream: [here](https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime/issues/110#issue-598275320)

**Comment by tpkessler** [comment]
Fixed with #103
